irctest/server_tests/isupport.py

```diff
@@ -9,6 +9,40 @@
 
 
 class IsupportTestCase(cases.BaseServerTestCase):
+    @cases.mark_specifications("Modern")
+    @cases.mark_isupport("PREFIX")
+    def testParameters(self):
+        """https://modern.ircdocs.horse/#rplisupport-005"""
+
+        # <https://modern.ircdocs.horse/#connection-registration>
+        # "Upon successful completion of the registration process,
+        #  the server MUST send, in this order:
+        #  [...]
+        #  5. at least one RPL_ISUPPORT (005) numeric to the client."
+        welcome_005s = [
+            msg for msg in self.connectClient("foo") if msg.command == "005"
+        ]
+        self.assertGreaterEqual(len(welcome_005s), 1)
+        for msg in welcome_005s:
+            # first parameter is the client's nickname;
+            # last parameter is a human-readable trailing, typically
+            # "are supported by this server"
+            self.assertGreaterEqual(len(msg.params), 3)
+            self.assertEqual(msg.params[0], "foo")
```

Comment:
```suggestion
            self.assertMessageMatch(
                msg,
                params=["foo", ANYSTR, ANYSTR, *ANYLIST],
            )
```

or

```suggestion
            self.assertMessageMatch(
                msg,
                params=["foo", ListRemainder(ANYSTR, min_length=3)],
            )
```

(plus the respective import from `irctest.patma`)